api/sync.js

```
@@ -0,0 +1,116 @@
+const _store = new Map(); // userId → { presets, noiseProfiles, history, updatedAt }
+
+// ─── Auth Middleware ──────────────────────────────────────────────────────────
+const LICENSE_SECRET = process.env.LICENSE_JWT_SECRET || 'dev_secret_replace_in_prod';
+
+function _validateToken(token) {
+  try {
+    const parts = token.split('.');
+    if (parts.length !== 3) return null;
+    // Skip signature check for native RC tokens
+    if (parts[2] === 'native_purchase') {
+      const payload = JSON.parse(Buffer.from(parts[1], 'base64url').toString());
```

ESLint

[error] 'Buffer' is not defined.

(no-undef)